DEVELOPMENT.md

```diff
@@ -0,0 +1,32 @@
+# Developing AccName
+Thanks for considering making a contribution to AccName! 🎉
+
+### Setup
+- Fork this repository.
+- Clone your fork to download a local copy.
+```
+$ git clone <your-fork-url>
+```
+- In the root folder of the project `accessible-name/`:
+```
+$ npm install
+```
+### Testing
+Tests are run using [Jasmine](https://jasmine.github.io/) and [Karma](https://karma-runner.github.io/latest/index.html).
+- To run tests, enter:
+```
+$ npm run test
+```
+- Any new features added to AccName should be appropriately tested with unit tests in a corresponding `_test.ts` file in the same directory as the edited file. For example, tests for `lib/ruleA.ts` should be in `lib/ruleA_test.ts`.
+### Code Formatting
+- [Google TypeScript Style](https://www.npmjs.com/package/gts) (GTS) is used to format the TypeScript code for AccName. The code is automatically formatted using a commit-hook. This means that the code is often automatically changed by GTS when it is committed, in which case you can simply `$ git add .` and `$ git commit --amend` to update your commit with the newly-formatted code.
+- AccName uses [JSDoc](https://jsdoc.app/) style comments.
+### Performance
```

Comment:
When I read performance I think speed/efficiency. Maybe 'regressions' or something is better? IDK it's also not great...

Reply:
How about 'Accuracy', 'Output Accuracy' or similar ?

Reply:
'Ouptut Accuracy' works :)